test/test_gpu_tensorcore.cpp

```diff
@@ -63,8 +63,11 @@ TEST_F(NVFuserTest, FusionAmpereMatmul_CUDA) {
   for (auto layout : kAllSupportedMmaLayout) {
     Fusion fusion;
     FusionGuard fg(&fusion);
-    auto tv0 = makeContigTensor(2, DataType::Half);
-    auto tv1 = makeContigTensor(2, DataType::Half);
+
+    auto shapes = matmulAtInputShape3DTuring(-1, -1, -1, layout);
```

Comment:
The special handling is not in `matmulAtInputShape3DTuring`. It is instead in `makeContigConcreteTensor`. `matmulAtInputShape3DTuring` will return shape like `[-1, 1, -1]`, and when `makeContigConcreteTensor`, it will know that this dim has symbolic shape which is not known at compile time.

Reply:
Thank you for explanation. I though so, but I wanted to double check this.